spec/menu_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'menu'
+
+describe Menu do
+
+  it "initialises a new instance of the Menu class" do
+    expect(subject).to be_a Menu
+  end
+
+  it "contains a list of dishes with prices" do
+    expect(subject.menu).to be_a Hash
+    expect(subject.menu["Fried rice"]).to be_a Numeric
+    expect(subject.menu).to have_key("Fried rice")
```

Comment:
This test ties you to a specific menu. You could improve your code by not hard-coding the menu: https://github.com/nboyse/takeaway-challenge/blob/master/docs/review.md#open-closed-principle